atom/browser/common_web_contents_delegate_mac.mm

```diff
@@ -22,17 +23,9 @@
   if (event.windowsKeyCode == ui::VKEY_ESCAPE && is_html_fullscreen())
     ExitFullscreenModeForTab(source);
 
-  BOOL handled = [[NSApp mainMenu] performKeyEquivalent:event.os_event];
-  if (!handled && event.os_event.window) {
-    // Handle the cmd+~ shortcut.
-    if ((event.os_event.modifierFlags & NSCommandKeyMask) /* cmd */ &&
-        (event.os_event.keyCode == 50  /* ~ */)) {
-      if (event.os_event.modifierFlags & NSShiftKeyMask) {
-        [NSApp sendAction:@selector(_cycleWindowsReversed:) to:nil from:nil];
-      } else {
-        [NSApp sendAction:@selector(_cycleWindows:) to:nil from:nil];
-      }
-    }
+  NSWindow* window = event.os_event.window;
+  if (window && [window isKindOfClass:[AtomNSWindow class]]) {
```

Comment:
@kevinsawicki The devtools window also needs the same handling code, otherwise it won't respond to `cmd+~`, steps to reproduce:

1. Open devtools window in detached state
1. Press `cmd+~` in it